components/financial_assistance/app/controllers/financial_assistance/applications_controller.rb

```diff
@@ -6,21 +6,52 @@ class ApplicationsController < FinancialAssistance::ApplicationController
 
     before_action :set_current_person
     before_action :set_family
-    before_action :find_application, :except => [:index, :index_with_filter, :new, :review, :raw_application]
+    before_action :find_application, :except => [:index, :index_with_filter, :new]
```

Comment:
The :review and :raw_application action methods already include queries to find the application. Removing these from the except list will result in querying the application twice, which can increase page load time.

I recommend either removing the application query from the respective methods or raising a ticket to address this refactor.

Reply:
Good catch. I will remove the duplicate queries.